### PR TITLE
Fix name of the test added by #32613

### PR DESCRIPTION
**Pull request by bogdanvlviv**
Related to https://github.com/rails/rails/pull/32613
r? @pixeltrix

### CODE REVIEW

File: activesupport/test/time_zone_test.rb [hunk 1]
```diff
@@ -725,7 +725,7 @@ def test_all_uninfluenced_by_time_zone_lookups_delegated_to_tzinfo
     assert_not_includes all_zones, galapagos
   end
 
-  def test_all_not_raises_exception_with_mizzing_tzinfo_data
+  def test_all_doesnt_raise_exception_with_missing_tzinfo_data
```
Comment (pixeltrix) [inline activesupport/test/time_zone_test.rb]: LOL, how did I miss that - thanks 👍

Comment (bogdanvlviv) [reply activesupport/test/time_zone_test.rb]: thanks for the review. will we backport this to `5-2-stable`?

Comment (pixeltrix) [reply activesupport/test/time_zone_test.rb]: Yes - I'll cherry pick

Comment (pixeltrix) [reply activesupport/test/time_zone_test.rb]: Done in e4ab3c17203cd90ec07ca78dc25b5989705bce16 - thanks again

**Pull request merged**